translations/ui-circulation/en_US.json

```diff
@@ -401,7 +401,7 @@
     "permission.settings.loan-history": "Settings (Circ): Can view loan history",
     "permission.settings.loan-policies": "Settings (Circ): Can create, edit and remove loan policies",
     "permission.settings.notice-policies": "Settings (Circ): Can create, edit and remove notice policies",
-    "permission.settings.circulation-rules": "Settings (Circ): Can create, edit and remove circulation rules",
+    "permission.settings.circulation-rules": "Settings (Circ): Can create, edit and remove circulation rules [LEGACY]",
```

Comment:
I copied this changed translation through to `en_US`, because otherwise the change in `en` is not visible. That's not necessary for the two translations that I added in `en`, as new translations shine through the `en_US` overlay.